opam.export

```diff
@@ -89,6 +89,9 @@ installed: [
   "caqti.1.5.0"
   "caqti-async.1.3.0"
   "caqti-driver-postgresql.1.5.1"
+  "caqti-driver-sqlite3.1.5.0"
```

Comment:
how did you figure out the versions in the end

Reply:
stopped using nix and regularly called `opam install ...`